tests/SDKTest.php

```diff
@@ -287,9 +288,20 @@ public function testHTTPSuccess()
                 
                 $this->assertEquals('GET:/v1/mock/tests/general/redirect/done:passed', $output[10]);
                 
-                if($options['supportUpload']) {
+                $supportsUpload = $options['supportUpload'];
+                if($supportsUpload) {
                     $this->assertEquals($output[11], 'POST:/v1/mock/tests/general/upload:passed');
                 }
+                
+                if($options['supportException']) {
```

Comment:
Maybe we can leave blank lines reserved for the upload test or even a comment in the code, otherwise every new feature here is going to get a really big block of if conditions.

Reply:
yes we need a way, may be simply print some value ('upload not supported yet') to make the outputs length and index consistent, so that we don't need multiple checks, let me know what you think?